<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QMainWindow" name="UpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>690</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Parameter Update</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MAC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="placeholderText">
       <string>CC:50:E3:9C:15:02</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UUID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_6">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>beb5483e-36e1-4688-b7f5-ea07361b26a8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Led Helligkeit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_7">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Probenanzahl</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>SAMPLEAVG_4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modus</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>MODE_MULTILED</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Abtastrate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_8">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>SAMPLERATE_200</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pulsweite</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>PULSEWIDTH_411</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ACDC Bereich</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>ADCRANGE_16384</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Update">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="Parameter_Update_Text">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGer_temanager">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionGer_temanager"/>
    <addaction name="Close"/>
   </widget>
   <addaction name="menuGer_temanager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMAC_und_UUID_ndern">
   <property name="text">
    <string>MAC und UUID ändern</string>
   </property>
  </action>
  <action name="actionGer_temanager">
   <property name="text">
    <string>MAC ändern</string>
   </property>
  </action>
  <action name="Close">
   <property name="text">
    <string>Schließen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Update</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close</sender>
   <signal>triggered()</signal>
   <receiver>UpdateWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close</sender>
   <signal>triggered()</signal>
   <receiver>progressBar</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
